server/data/converters/trivyScans.ts

```diff
@@ -1,11 +1,9 @@
 /* eslint-disable camelcase */
 import { TrivyScan } from '../strapiApiTypes'
 import type { ScanSummary, TrivyScanType } from '../modelTypes'
-import { DataItem } from '../strapiClientTypes'
 
-const convertTrivyScan = (trivyScan: DataItem<TrivyScan>): TrivyScanType => {
-  const { attributes, id } = trivyScan
-  const { name, trivy_scan_timestamp, build_image_tag, scan_status, scan_summary, environments } = attributes
+const convertTrivyScan = (trivyScan: TrivyScan): TrivyScanType => {
```

Comment:
can probably just remove this entirely and `TrivyScanType` but probably better as a follow up

Reply:
(as we're just copying the same fields across)